spec/controllers/trainer_controller_spec.rb

```diff
@@ -146,6 +157,28 @@
         card = check_review_card(card, 'car', 3)
         expect(card.review_attempt).to eq(2)
       end
+
+      it 'set review_count try=4' do
+        card = create_and_check_review_card(@user, @block, 3, 'car')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.